### PR TITLE
PXC-3480 garbd documentation does not instruct on SSL requirement (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M  source/howtos/garbd_howto.rst

### CODE REVIEW

File: doc/source/howtos/garbd_howto.rst [hunk 1]
```diff
@@ -1,60 +1,66 @@
...
 
    root@ubuntu:~# apt install percona-xtradb-cluster-garbd
 
-on Debian/Ubuntu distributions, or:
+*Galera Arbitrator* can be installed from Percona's repository on RedHat or derivative distributions with the following command:
 
 .. code-block:: bash
 
-   [root@centos ~]# yum install percona-xtradb-cluster-garbd
+   [root@centos ~]# yum install percona-xtradb-cluster-garbd.
```
Comment (satya-bodapati) [inline doc/source/howtos/garbd_howto.rst]: Remove . at the end

File: doc/source/howtos/garbd_howto.rst [hunk 2]
```diff
@@ -1,60 +1,66 @@
...
-<http://galeracluster.com/documentation-webpages/arbitrator.html>`
-is a member of |Percona XtraDB Cluster| that is used for voting
-in case you have a small number of servers (usually two)
-and don't want to add any more resources.
-Galera Arbitrator does not need a dedicated server.
-It can be installed on a machine running some other application.
-Just make sure it has good network connectivity.
+A `Galera Arbitrator
+<https://galeracluster.com/library/documentation/arbitrator.html>`__ 
+is a lightweight member of a **Percona XtraDB Cluster**. This member can vote but does not do any replication and is not included in flow control calculations. The Galera Arbitrator is a separate daemon called ``garbd``. You can start this node separately from the cluster. You cannot configure this node using the ``my.cnf`` file. You can  run this deamon either as a service or from the shell.
```
Comment (satya-bodapati) [inline doc/source/howtos/garbd_howto.rst]: remove extra space before 'run'

File: doc/source/howtos/garbd_howto.rst [hunk 3]
```diff
@@ -89,16 +98,11 @@ up. This example is using cluster information from the :ref:`ubuntu_howto`.
 
   # Optional Galera internal options string (e.g. SSL settings)
   # see http://galeracluster.com/documentation-webpages/galeraparameters.html
-  # GALERA_OPTIONS=""
+  # GALERA_OPTIONS="socket.ssl_cert=/etc/ssl/mysql/server-key.pem;socket.ssl_key=/$"
```
Comment (satya-bodapati) [inline doc/source/howtos/garbd_howto.rst]: ssl_key is empty, should it use /etc/ssl/mysql/server-key.pem

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: doc/source/howtos/garbd_howto.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/howtos/garbd_howto.rst]: Done.

File: doc/source/howtos/garbd_howto.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/howtos/garbd_howto.rst]: Done.

File: doc/source/howtos/garbd_howto.rst (hunk 2 above)
Comment (alina-derkach-oaza) [inline doc/source/howtos/garbd_howto.rst]: deamon -> daemon

Comment (patrickbirch) [reply doc/source/howtos/garbd_howto.rst]: Fixed.

File: doc/source/howtos/garbd_howto.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/howtos/garbd_howto.rst]: Done

File: doc/source/howtos/garbd_howto.rst [hunk 4]
```diff
@@ -1,60 +1,66 @@
...
+is a lightweight member of a **Percona XtraDB Cluster**. This member can vote but does not do any replication and is not included in flow control calculations. The Galera Arbitrator is a separate daemon called ``garbd``. You can start this node separately from the cluster. You cannot configure this node using the ``my.cnf`` file. You can  run this deamon either as a service or from the shell.
 
-Galera Arbitrator is a member of the cluster that participates in the voting,
-but not in actual replication
-(although it receives the same data as other nodes).
-Also, it is not included in flow control calculations.
-
-This document will show how to add Galera Arbitrator node
-to an existing cluster.
+This document will show how to add Galera Arbitrator node to an existing cluster.
```
Comment (alina-derkach-oaza) [inline doc/source/howtos/garbd_howto.rst]: This document SHOWS how to add a Galera Arbitrator node to an existing cluster.

File: doc/source/howtos/garbd_howto.rst [hunk 5]
```diff
@@ -1,60 +1,66 @@
...
-   [root@centos ~]# yum install percona-xtradb-cluster-garbd
+   [root@centos ~]# yum install percona-xtradb-cluster-garbd.
 
-on CentOS/RHEL distributions.
+Starting ``garbd`` and configuration
+=========================================
 
-Configuration
-=============
+For this document, we use the cluster information from :ref:`ubuntu_howto`.
```
Comment (alina-derkach-oaza) [inline doc/source/howtos/garbd_howto.rst]: In this document?

File: doc/source/howtos/garbd_howto.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/howtos/garbd_howto.rst]: Rewritten

File: doc/source/howtos/garbd_howto.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/howtos/garbd_howto.rst]: Removed

**Review by satya-bodapati — APPROVED** [review]

File: doc/source/howtos/garbd_howto.rst [hunk 6]
```diff
@@ -1,60 +1,68 @@
...
 
-`Galera Arbitrator
-<http://galeracluster.com/documentation-webpages/arbitrator.html>`
-is a member of |Percona XtraDB Cluster| that is used for voting
-in case you have a small number of servers (usually two)
-and don't want to add any more resources.
-Galera Arbitrator does not need a dedicated server.
-It can be installed on a machine running some other application.
-Just make sure it has good network connectivity.
+The size of a cluster increases when a node joins the cluster and decreases when a node leaves. The nodes how a cluster reacts to replication problems with inconsistency voting. The size of the cluster determines the required votes to achieve a quorum. If a node no longer responds and is disconnected from the cluster the remaining nodes vote. The majority of the nodes that vote are considered to be in the cluster.
```
Comment (alina-derkach-oaza) [inline doc/source/howtos/garbd_howto.rst]: The nodes how a cluster reacts to replication problems with inconsistency voting. - This sentence seems to be incomplete.

Comment (patrickbirch) [reply doc/source/howtos/garbd_howto.rst]: Fixed.

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**